replica/table.cc

```diff
@@ -858,7 +858,11 @@ table::stop() {
         co_return;
     }
     co_await _async_gate.close();
-    co_await await_pending_ops();
+    // When dropping a table, ops phasers are closed when detaching.
+    if (!_pending_reads_phaser.is_closed()) {
```

Comment:
I'll be more comfortable if we rely on a table member to decide if we skip this than non a phaser internal state.